File: python/sort/bubblesort.py
def BUBBLE_SORT(A):
    n = len(A)
    for i in range(n):
        for j in range(n-1, i-1, -1):
            if A[j] < A[j-1]:
                A[j], A[j-1] = A[j-1], A[j]
    return A

A = BUBBLE_SORT(map(int, raw_input().split()))
print A

File: java/sort/BubbleSort.java
class BubbleSort
{
  public static void bubbleSort(int a[])
  {
    for(int i=0;i<a.length-1;i++)
    {
      for(int j=0;j<a.length-1-i;j++)
      {
        if(a[j]>=a[j+1])
        {
          int t=a[j];
          a[j]=a[j+1];
          a[j+1]=t;
        }
      }
    }
  }
}
